user/api.go

```diff
@@ -742,13 +751,14 @@ func (a *Api) Login(res http.ResponseWriter, req *http.Request) {
 			// Default role to patient, if no role is found
 			// FIXME Dirty quirk
 			result.Roles = []string{"patient"}
+			a.logger.Debugf("add default role to patient for username: %s", result.Username)
 		}
 		if requestSource == "private" && !result.HasRole("patient") {
-			a.logger.Printf("Private route login: Adding patient role to user %v", result.Id)
+			a.logger.Debugf("Private route login: Adding patient role to user %v", result.Id)
 			// Let's add the role patient:
 			result.Roles = append([]string{"patient"}, result.Roles...)
 			if err := a.Store.UpsertUser(req.Context(), result); err != nil {
-				a.logger.Printf("Login of a non patient user from our private endpoint. Error while adding the role patient: %s", err)
+				a.logger.Debugf("Login of a non patient user from our private endpoint. Error while adding the role patient: %s", err)
```

Comment:
I would raise log level to Warn here we have a mogodb error, we should see it (I think that default global log_level set in helm config is info)